### PR TITLE
test: [M3-6225] - Clean up cy.intercept calls in nodebalancer

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝
Clean up cy.intercept calls in nodebalancer

## Changes  🔄
List any change relevant to the reviewer.

- Replace calls to cy.intercept with intercept utils in nodebalancers.ts.
- Adds interceptCreateNodeBalancers() util

## How to test 🧪
We can rely on CI to confirm that this test has not been broken by these changes, but to run the test locally you can use this command:
`yarn cy:run -s "cypress/e2e/core/nodebalancers/smoke-create-nodebal.spec.ts"`

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [ ] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [x] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/cypress/support/intercepts/nodebalancers.ts [hunk 1]
```diff
@@ -23,3 +23,14 @@ export const mockGetNodeBalancers = (
...
 };
+
+/**
+ * Intercepts POST request to intercept nodeBalancer data.
+ *
+ * @param
+ *
+ * @returns Cypress chainable.
+ */
+export const interceptCreateNodeBalancers = (): Cypress.Chainable<null> => {
```
Comment (jdamore-linode) [inline packages/manager/cypress/support/intercepts/nodebalancers.ts:34]:
```suggestion
export const interceptCreateNodeBalancer = (): Cypress.Chainable<null> => {
```
Extremely minor nit: still holding out hope that some day there will be batch create APIs where we can create multiple resources at once 😅

File: packages/manager/cypress/support/intercepts/nodebalancers.ts [hunk 2]
```diff
@@ -23,3 +23,14 @@ export const mockGetNodeBalancers = (
     paginateResponse(nodeBalancers)
   );
 };
+
+/**
+ * Intercepts POST request to intercept nodeBalancer data.
+ *
+ * @param
```
Comment (jdamore-linode) [inline packages/manager/cypress/support/intercepts/nodebalancers.ts:30]:
```suggestion
```

**Review by jdamore-linode — APPROVED** [review]
Awesome, great work @AzureLatte! 🎉 Don't forget to push your changesets that were created with `yarn changeset`

**Review by dwiley-akamai — APPROVED** [review]
Congrats on your first PR @AzureLatte!

`smoke-create-nodebal.spec.ts` passes ✅ 

Approved pending Joe's feedback being addressed

**Comment by AzureLatte** [comment]
For failing tests, 

- Rebuild test failed because of a 502 response from the API when getting LKE clusters
- StackScript test is known to be broken due to an Image deprecation that was made in the backend yesterday; we have a PR open to fix it

**Pull request merged**